packages/running/src/index.tsx

```diff
@@ -242,27 +231,15 @@ function Item(props: {
   if (runningItem.className) {
     classList.push(runningItem.className);
   }
-  if (props.child) {
-    classList.push('jp-mod-running-child');
-  }
-  if (props.child && !children) {
-    classList.push('jp-mod-running-leaf');
-  }
 
   return (
     <>
-      <li>
-        <div
-          className={classList.join(' ')}
-          onClick={onClick}
-          data-context={runningItem.context || ''}
-        >
```

Comment:
You could try to restore that div with only the className to see if it restores the switch between tree view and list view.

Reply:
@fcollonval It is not a good way to do this after doing these changes the problem is not solve and the ui is looking bad . 

https://github.com/jupyterlab/jupyterlab/assets/145124929/ca837ec5-2f2b-4f94-acfc-4b43430255dd